forge.config.ts

```diff
@@ -0,0 +1,75 @@
+import type { ForgeConfig } from "@electron-forge/shared-types";
+import { MakerSquirrel } from "@electron-forge/maker-squirrel";
+import { MakerZIP } from "@electron-forge/maker-zip";
+import { MakerDeb } from "@electron-forge/maker-deb";
+import { MakerRpm } from "@electron-forge/maker-rpm";
+import { MakerFlatpak } from "@electron-forge/maker-flatpak";
+import { MakerSnap } from "@electron-forge/maker-snap";
+import { VitePlugin } from "@electron-forge/plugin-vite";
+
+const config: ForgeConfig = {
+    packagerConfig: {
+        asar: false, // Cannot enable asar, it breaks map parsing
+    },
+    rebuildConfig: {},
+    makers: [
+        new MakerSquirrel({}),
+        new MakerZIP({}, ["darwin"]),
+        new MakerRpm({
+            options: {
+                mimeType: ["application/sdfz"],
+                categories: ["Game"],
+                icon: "src/renderer/assets/images/icon.png",
+                homepage: "https://www.beyondallreason.info/",
+                license: "MIT",
+                name: "bar-lobby",
+                productName: "BAR Lobby",
+            },
+        }),
+        new MakerDeb({}),
+        new MakerFlatpak({}),
+        new MakerSnap({}),
```

Comment:
I would start only with appimage and windows installer.

Reply:
We need a format for Linux that is not tied to any distributions. I see in that thread that there are some third-party packagers for appimage.

I also see that electron forge points at electron updater which points at https://www.electronjs.org/docs/latest/api/auto-updater#platform-notices for autoupdates

> There is no built-in support for auto-updater on Linux

so, using electron forge might be dead on arrival

Reply:
Need some thinking.

But even if we decide to go back to electron-builder it shouldn't be a big deal and it can be decided later.